content/compose/compose-file/05-services.md

```diff
@@ -1154,7 +1147,7 @@ There is a performance penalty for applications that swap memory to disk often.
 
 ## network_mode
 
-`network_mode` sets a service container's network mode. Available values are platform specific, but Compose defines specific values which must be implemented as described if supported:
+`network_mode` sets a service container's network mode. 
```

Comment:
Is `service:{name}` the same as a container network described here? https://docs.docker.com/network/#container-networks
Where the container gets the networking stack of another container.
Would it be useful to expand and/or add a link?